<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatServerForm</class>
 <widget class="QWidget" name="ChatServerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>351</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="waitingRoomLabel">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>tr(&quot;waitting room&quot;)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeWidget" name="waittingRoomTreeWidget">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <column>
            <property name="text">
             <string>tr(&quot;Client status&quot;)</string>
            </property>
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>11</pointsize>
              <bold>true</bold>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string>tr(&quot;Client list&quot;)</string>
            </property>
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>11</pointsize>
              <bold>true</bold>
             </font>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_1">
         <item>
          <widget class="QLabel" name="chattingRoomLabel">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>tr(&quot;chatting room&quot;)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeWidget" name="chattingRoomTreeWidget">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <column>
            <property name="text">
             <string>tr(&quot;Client status&quot;)</string>
            </property>
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>11</pointsize>
              <bold>true</bold>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string>tr(&quot;Client list&quot;)</string>
            </property>
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>11</pointsize>
              <bold>true</bold>
             </font>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="resetPushButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>tr(&quot;ClientListReset&quot;)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="logLabel">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>tr(&quot;chatting log&quot;)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="logTreeWidget">
         <column>
          <property name="text">
           <string>IP</string>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>tr(&quot;Port&quot;)</string>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>tr(&quot;ID&quot;)</string>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>tr(&quot;name&quot;)</string>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>tr(&quot;message&quot;)</string>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>tr(&quot;time&quot;)</string>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <bold>true</bold>
           </font>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="clearPushButton">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>tr(&quot;clear&quot;)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="savePushButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>tr(&quot;save&quot;)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clearPushButton</sender>
   <signal>clicked()</signal>
   <receiver>logTreeWidget</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>662</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>744</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
